File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3-Feb_Solutions_Three/C++/prog5.cpp
#include<iostream>

int main() {
	long n, sum = 0, num = 0;
	std::cout << "Enter elemet no : ";
	std::cin >> n;
	for(int i = 0; i < n; i++) {
		num = num * 10 + 9;
		sum += num;
		if(i != n - 1) {
			std::cout << num <<" + ";
		} else {
			std::cout << num << " = ";
		}
	}
	std::cout << sum;
	return 0;
}

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3-Feb_Solutions_Three/Java/prog5.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter n : ");
		int n = sc.nextInt();
		long num = 0, sum = 0;
		
		for(int i = 0; i < n; i++) {
			num = num * 10 + 9;
			sum += num;
			if(i != n - 1) {
				System.out.print(num + " + ");
			} else {
				System.out.print(num + " = ");
			}
		}

		System.out.println(sum);

	}
}
